codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/smithy/generators/client/CustomizableOperationGenerator.kt

```diff
@@ -23,8 +26,17 @@ class CustomizableOperationGenerator(
     private val generics: FluentClientGenerics,
     private val includeFluentClient: Boolean,
 ) {
+    companion object {
+        const val CUSTOMIZE_MODULE = "crate::operation::customize"
+    }
+
     fun render(crate: RustCrate) {
-        crate.withModule(FluentClientGenerator.customizableOperationModule) { writer ->
+        crate.withModule(RustModule.Operation) { writer ->
+            writer.docs("Operation customization and supporting types")
+            writer.rust("pub mod customize;")
+        }
```

Comment:
It would be cool if this could be handled automatically by `withNonRootModule` somehow but I guess that would open us up to annoying codegen write order issues.

Reply:
Certainly an area for improvement. I think we can improve on this incrementally.